source/develop/release-management/features/sla/hosted-engine-new-deployment.html.md

```diff
@@ -0,0 +1,150 @@
+16. Add a disk for Hosted Engine lockspace
+17. Add a disk for Hosted Engine metadata
+18. Add a disk for Hosted Engine VM 
+19. Add the Hosted Engine VM
+20. Clean the /etc/hosts file on the VM
+21. Configure libvirt to listen to TLS and restart it
+23. Destroy the bootstrap VM 
+23. Undefine the bootstrap VM
+24. Create the configuration files: vm.conf, broker.conf, version, fheanswers.conf, hosted-engine.conf
+25. Create the configuration archive 
+26. Create /var/run/ovirt-hosted-engine-ha/ and copy vm.conf there  
+27. Find the storage domain mount point 
+28. Copy the archive to the configuraion disk 
+29. Copy the bootstrap vm disk to the vm disk
+30. Clean the /etc/hosts file on the host
+31. Start the broker and the agent services  
+
+## Vars file: 
+
+FQDN: myvm.mydomain - FQDN where the hosted engine vm shoud run   
+VM_MAC_ADDR: 00:16:3e:1e:15:d7 - mac address for the VM   
+CLOUD_INIT_DOMAIN_NAME: mydomain - domain name for the VM  
+CLOUD_INIT_HOST_NAME: myvm - host name for the VM   
+HOST_NAME: sansa - host name for the Hosted Engine host, this will appear in the UI  
+HOST_ADDRESS: sansa.usersys.redhat.com - host address for the Hosted Engine host  
+LOCAL_VM_DIR: /usr/localvm - the local directory on the host where the all the configuration files will be created   
+ADMIN_PASSWORD: 1234 - password for the engine   
+APPLIANCE_PASSWORD: qum5net - password for the Hosted Engine VM   
+HOST_PASSWORD: qum5net - password for the Hosted Engine host   
+STORAGE_DOMAIN_NAME: he_storage - desired storage domain name   
+TIME_ZONE: Asia/Jerusalem - timezone for the VM   
+BRIDGE: ovirtmgmt - management bridge name   
+VM_NAME: HostedEngine - Hosted Engine VM name, best not to change it as some features depend on that name   
+MEM_SIZE: 2048 - memory size for the VM  
+CONSOLE_TYPE: vnc - console type   
+SP_UUID: 00000000-0000-0000-0000-000000000000 - storage pool ID, I used the default storage pool   
+CDROM_UUID:
+CDROM:
+NIC_UUID: 45645645 
+CONSOLE_UUID:
+VIDEO_DEVICE: vga - video device   
+GRAPHICS_DEVICE: vnc - graphics device  
+VCPUS: 4 - virtual cpus for the VM  
+MAXVCPUS: 4 - max virtual CPUs for the VM   
+CPU_SOCKETS: 2 - CPU sockets   
+CPU_TYPE: Conroe - CPU type   
+EMULATED_MACHINE: pc  - emulated machine   
+VERSION: 2.2.0-0.0.master.20170413122212 - the version for the version file in the configuration archive
+GATEWAY: 10.35.1.254 - gateway 
+DOMAIN_TYPE: nfs3 - desired domain type 
+ISCSI_PORT: 3260 
+ISCSI_TARGET: iqn.2017-10.com.redhat.tlv:sla-leonard
+LUN_ID: 36001405879c330daed74eae8ec28a237 - ansible module requires lun id   
+ISCSI_STORAGE: false - whether this is iscsi domain or not      
+ISCSI_USERNAME: username  
+ISCSI_PASSWORD: password  
+STORAGE: shablul.eng.lab.tlv.redhat.com:/exports/multipass/export/images/rnd/etokar/sansa/he/exports/multipass/export/images/rnd/etokar/sansa/he - path to the storage, in ISCSI case this is the IP of the portal     
```

Comment:
@JohnMarksRH care to remove DNS names (ie- replace shablul.eng.lab.tlv.redhat.com with example.com)?